eng/Versions.props

```diff
@@ -30,4 +30,9 @@
     <MicrosoftNETCoreDotNetHostPackageVersion>3.0.0-preview-27314-4</MicrosoftNETCoreDotNetHostPackageVersion>
     <MicrosoftNETCoreDotNetHostPolicyPackageVersion>3.0.0-preview-27314-4</MicrosoftNETCoreDotNetHostPolicyPackageVersion>
   </PropertyGroup>
+  <!-- Coreclr dependencies -->
+  <PropertyGroup>
```

Comment:
Nit: Now that you're doing this. Would you mind moving this properties to the property group above? Including the comment as we did with the core-setup ones.

Reply:
Done